examples/balrog/babyai/README.md

```diff
@@ -0,0 +1,111 @@
+# Example: LLM applicationfor BabyAI
+
+<!-- TensorZero was built to support inference strategies more sophisticated than just a single chat completion.
+
+In this example, we'll show how you can drop in this experimental best-of-n variant type to spend additional compute budget for better performance on a challenging LLM benchmark. -->
```

Comment:
do we actually have a best-of-n in this example? I think it would actually be quite interesting to add that.

Reply:
nvm this is a comment, duh. We should cut this (or actually add mixture of N)